card/lib/cardio.rb

```diff
@@ -18,6 +18,10 @@ module Cardio
   mattr_reader :paths, :config, :cache
 
   class << self
+    def cache
+      @@cache ||= ::Rails.cache
+    end
+    
```

Comment:
I get that we don't really want this dependency long term, but for now this avoids problems.